internal/server/config.go

```diff
@@ -1,8 +1,12 @@
 package server
 
 type Config struct {
```

Comment:
Can you make changes to [`config/config.yaml`](https://github.com/odpf/shield/blob/rc/0.5.1/config/config.yaml) as well?